tests/BenchmarkDotNet.Tests/ParameterInstanceTests.cs

```diff
@@ -15,18 +17,20 @@ public class ParameterInstanceTests
         [InlineData("short")]
         public void ShortParameterValuesDisplayOriginalValue(object value)
         {
-            var parameter = new ParameterInstance(definition, value);
+            var config = DefaultConfig.Instance.CreateImmutableConfig();
+            var parameter = new ParameterInstance(definition, value, config);
```

Comment:
It is not an issue. Just my opinion.

I think the `ParameterInstance` class doesn't need the whole configuration. `config.SummaryStyle` would be enough (or maybe even `config.SummaryStyle?.MaxParamterColumnWidth`). I would create an `ISummaryStyle` interface.

Currently, this test is not a unit test because you test two elements. Creating the default configuration and `ParameterInstance.ToDisplayText()`

Reply:
Good point, I've made `ParameterInstance` depend on `SummaryStyle`, not entire `Config`